dev/integration_tests/external_ui/test_driver/main_test.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+final RegExp statsRegExp = new RegExp('Produced: (.*)fps\nConsumed: (.*)fps\nWidget builds: (.*)');
+
+Future<Null> main() async {
+
+  // TODO(mravn): the following pause appears necessary on iOS to avoid
+  // inflating elements too early (when there is no size).
+  await new Future<Null>.delayed(const Duration(seconds: 1));
```

Comment:
Should this pause be injected after `FlutterDriver.connect()`? It seems that if placed here, this pause will be racing other initialization activities that are happening prior to booting the UI thread. But if placed _after_ the connection is established we will know for sure that the UI thread is alive and responding.